lib/rspec/support/ruby_features.rb

```diff
@@ -103,88 +60,20 @@ def supports_taint?
           true
         end
       end
-      ripper_requirements = [ComparableVersion.new(RUBY_VERSION) >= '1.9.2']
-
-      ripper_requirements.push(false) if Ruby.rbx?
-
-      if Ruby.jruby?
-        ripper_requirements.push(Ruby.jruby_version >= '1.7.5')
-        # Ripper on JRuby 9.0.0.0.rc1 - 9.1.8.0 reports wrong line number
-        # or cannot parse source including `:if`.
-        # Ripper on JRuby 9.x.x.x < 9.2.1.0 can't handle keyword arguments.
-        ripper_requirements.push(!Ruby.jruby_version.between?('9.0.0.0.rc1', '9.2.0.0'))
-      end
 
-      if ripper_requirements.all?
-        def ripper_supported?
-          true
-        end
-      else
+      # Ripper on JRuby 9.0.0.0.rc1 - 9.1.8.0 reports wrong line number
+      # or cannot parse source including `:if`.
+      # Ripper on JRuby 9.x.x.x < 9.1.17.0 can't handle keyword arguments
+      # Neither can JRuby prior to 9.2.1.0
+      if Ruby.rbx? || (Ruby.jruby? && JRUBY_VERSION < '9.2.1.0')
```

Comment:
@JonRowe, do we keep Rubinius support, and this check here, and just dropping it from our CI matrix? Or should I remove this check, and only keep JRuby check?

Reply:
Keep the check, and just drop from our CI matrix. If someone comes to us with Rubinius or Truffle problems we will try to help, just not going to run CI agains them, put back the call to `jruby_version` though :)